app/src/organisms/Desktop/Devices/ProtocolRun/ProtocolRunHeader/RunHeaderContent/RunHeaderSectionUpper.tsx

```diff
@@ -74,7 +85,19 @@ export function RunHeaderSectionUpper(
             {t('visualize')}
           </SecondaryButton>
         ) : null}
-        <ActionButton {...props} />
+        <Flex gridGap={SPACING.spacing8}>
+          {isCancellableStatus(runStatus) && (
+            <PrimaryButton
```

Comment:
```suggestion
            <AlertPrimaryButton
```
Figma does say the component is a `PrimaryButton`, but figma doesn't reflect our code when it comes to these older desktop buttons. It's a fun pitfall that AUTH is nice enough to be addressing currently.

We typically use the `AlertPrimaryButton` whenever we need a red button, because this gives us all our accessibility states (such as hover) for free.